Source/Events.Processing/Streams/CreateScopedStreamProcessors.cs

```diff
@@ -16,21 +17,29 @@ namespace Dolittle.Runtime.Events.Processing.Streams
     public class CreateScopedStreamProcessors : ICreateScopedStreamProcessors
     {
         readonly IEventFetchers _eventFetchers;
-        readonly IStreamProcessorStateRepository _streamProcessorStates;
+        readonly IResilientStreamProcessorStateRepository _streamProcessorStates;
+        readonly IPolicies _policies;
         readonly ILoggerManager _loggerManager;
         readonly TenantId _tenant;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="CreateScopedStreamProcessors"/> class.
         /// </summary>
         /// <param name="eventFetchers">The <see cref="IEventFetchers" />.</param>
-        /// <param name="streamProcessorStates">The <see cref="IStreamProcessorStateRepository" />.</param>
+        /// <param name="streamProcessorStates">The <see cref="IResilientStreamProcessorStateRepository" />.</param>
         /// <param name="executionContextManager">The <see cref="IExecutionContextManager" />.</param>
+        /// <param name="policies">The <see cref="IPolicies" />.</param>
         /// <param name="loggerManager">The <see cref="ILoggerManager" />.</param>
-        public CreateScopedStreamProcessors(IEventFetchers eventFetchers, IStreamProcessorStateRepository streamProcessorStates, IExecutionContextManager executionContextManager, ILoggerManager loggerManager)
+        public CreateScopedStreamProcessors(
+            IEventFetchers eventFetchers,
+            IResilientStreamProcessorStateRepository streamProcessorStates,
+            IExecutionContextManager executionContextManager,
+            IPolicies policies,
```

Comment:
Why not `IAsyncPolicyFor<ICanFetchEventsFromStream>`?

Reply:
https://github.com/dolittle-runtime/Runtime/pull/408/commits/554add08e459ccc031019dcbde7d23b4f6928a23